larq/quantizers.py

```diff
@@ -74,17 +74,13 @@
 ]
 
 
-@tf.custom_gradient
-def _binarize_with_identity_grad(x):
-    return math.sign(x), lambda dy: dy
-
-
-@tf.custom_gradient
-def _binarize_with_weighted_grad(x):
-    def grad(dy):
-        return (1 - tf.abs(x)) * 2 * dy
+def _clipped_gradient(x, dy, clip_value):
+    if clip_value is None:
+        return dy
 
-    return math.sign(x), grad
+    zeros = tf.zeros_like(dy)
+    mask = tf.math.less_equal(tf.math.abs(x), clip_value)
+    return tf.where(mask, dy, zeros)
```

Comment:
We could also explicitly call `where_v2` here, but I think that makes it a lot more readable.

Reply:
API is the same right? If `where_v2` is faster/recommended I would use that.

Reply:
I think `where` and `where_v2` are equally readable so lets go for whichever one is faster.

Reply:
`tf.where == tf.where_v2 == tf.compat.v2.where` on TensorFlow 2.0.0. Unfortunately it is not available in TF 1.13 so it would be messy to use it. This comment was mostly for reference.